lib/PhpStatistics.php

```diff
@@ -64,4 +66,67 @@ protected function cleanVersion($version) {
 		}
 		return $version;
 	}
+
+	/**
+	 * Get status information about the cache from the OPcache extension
+	 *
+	 * @return array with an array of state information about the cache instance
+	 */
+	protected function getOPcacheStatus() {
+		// Test if the OPcache module is installed
+		if (!extension_loaded('Zend OPcache')) {
+			// module not loaded
+			// returning back zeroes to prevent any errors on JS side.
+			return 'N/A';
+		}
+
+		// get status information about the cache
+		$status = opcache_get_status(false);
```

Comment:
If plesk is used this method is disabled by default.

Reply:
I could add a check for this by using https://www.php.net/manual/en/function.function-exists.php. What do you think? I guess we can't do anything about this plesk issue. Most people confronted with this problem will probably not have the permissions to change this.

Reply:
I think it's fine. The method seems to return false if disabled.